src/Microsoft.AspNet.Mvc.Localization/IViewLocalizer.cs

```diff
@@ -4,7 +4,7 @@
 namespace Microsoft.AspNet.Mvc.Localization
 {
     /// <summary>
-    /// A service that provides localized strings for views.
+    /// Represents a type that provides HTML-aware localization for views.
```

Comment:
This is standard .NET API reference docs language, e.g. https://msdn.microsoft.com/en-us/library/s4ys34ea(v=vs.110).aspx